Update `process` from v0.5.x to v0.11.x

shinnn opened the pull request:
npm CLI >= v3.0.0 tries to dedupe the dependencies by default, and keeping dependencies up-to-date helps better deduplication.

shinnn commented:
@Raynos @Matt-Esch What do you think?

snuggs commented:
@Raynos @shinnn @Matt-Esch please update this branch and merge in if applicable.

honzajavorek commented:
@Raynos The old `process` version is missing a full license text and thus depending on it is problematic with most company lawyers. The newer version does not suffer the problem anymore, so it would be awesome if you updated the version and released a new version of `global`, which could propagate the change up to all dependents.

Things like Storybook are depending on this library, so it would be nice to sort it out.

shinnn committed:
Update `process` from v0.5.x to v0.11.x

npm CLI >= v3.0.0 tries to dedupe the dependencies by default, and keeping dependencies up-to-date helps better deduplication.

shinnn commented:
Friendly ping @Raynos.

Raynos commented:
Published 4.4.0

shinnn commented:
@Raynos Thanks. I appreciate you maintain this project [over 6 years](https://github.com/Raynos/global/commit/01b936607bc90ea705badd0ab56154303a6f48e9).

Raynos commented:
Correction, I maintained this project in 2014 and 2019 ... 

In 2017 & 2018 I was bad at open source :disappointed:
